content/docs/guides/multi-target-exporter.md

```diff
@@ -0,0 +1,448 @@
+  prom/prometheus \
+  --config.file="/prometheus.yml"
+```
+
+Run Prometheus on MacOS and Windows:
+
+```bash
+docker \
+  run -p 9090:9090 \
+  --mount type=bind,source="$(pwd)"/prometheus.yml,target=/prometheus.yml,readonly \
+  prom/prometheus \
+  --config.file="/prometheus.yml"
+```
+
+This command works similarly to [running the blackbox exporter using a config file](#run-exporter).
+
+If everything worked, you should be able to go to [localhost:9090/targets](http://localhost:9090/targets) and see under `blackbox-exporter-metrics` an endpoint with the state `UP` in green. If you get a red `DOWN` make sure that the blackbox exporter you started [above](run-exporter) is still running. If you see nothing or a yellow `UNKNOWN` you are really fast and need to give it a few more seconds before reloading your browser’s tab.
+
+To tell Prometheus to query `"localhost:9115/probe?target=prometheus.io&module=http_2xx"` you add another scrape job where you set the `metrics_path` to `/probe` and the parameters under `params:` in the Prometheus config file `prometheus.yml`:
+
+<a name="prometheus-config"></a>
+
+```yaml
+scrape_configs:
+
+- job_name: blackbox-exporter-metrics
+  scrape_interval: 5s
+  metrics_path: /metrics
+  static_configs:
+    - targets:
+      - localhost:9115   # for Windows and macOS replace with - host.docker.internal:9115
+
+- job_name: 'blackbox'
+  scrape_interval: 5s
+  metrics_path: /probe
+  params:
+    module: [http_2xx]
+    target: [prometheus.io]
+  static_configs:
+    - targets:
+      - localhost:9115   # for Windows and macOS replace with - host.docker.internal:9115
+```
+
+After saving the config file switch to the terminal with your Prometheus docker container and stop it by pressing `ctrl+C` and start it again to reload the configuration by using the existing [command](#run-prometheus).
+
+The terminal should return the message `"Server is ready to receive web requests."` and after a few seconds you should start to see colourful graphs in [your Prometheus](http://localhost:9090/graph?g0.range_input=5m&g0.stacked=0&g0.expr=probe_http_duration_seconds&g0.tab=0).
+
+This works, but it has a few disadvantages:
+
+1. The actual targets are up in the param config, which is very unusual and hard to understand later.
+1. The `instance` label has the value of the blackbox exporter’s address which is technically true, but not what we are interested in.
+1. We can’t see which URL we probed. This is unpractical and will also mix up different metrics into one if we probe several URLs.
+
+To fix this, we will use [relabeling](/docs/prometheus/latest/configuration/configuration/#<relabel_config>).
+Relabeling is very powerful for advanced usecases because behind the scenes many things in Prometheus are configured with internal labels.
+The details are worthy of a [talk](https://www.youtube.com/watch?v=b5-SvvZ7AwI) on its own. For now is suffices if you understand this:
```

Comment:
Maybe rephrase that the the details are complicated, outside of scope, you will limit yourself to what's needed here and then refer to the talk as further info?